python/plugins/per-project-location/README.rst

```diff
@@ -0,0 +1,22 @@
+..
```

Comment:
Shouldn't this be an .md file in order to adhere to the new policy?

Reply:
nice catch.
I'll update .